app/services/idv/actions/cancel_update_address_action.rb

```diff
@@ -0,0 +1,9 @@
+module Idv
```

Comment:
Do we need this action for the existing DocAuthFlow ? It doesn't appear that we included any changes to call it, and I suspect we wouldn't need them, since `address` is not a real step in the flow.

Reply:
The ERB template uses the current flow's version of the action, so this will be called if the "Back" link is selected while using the DocAuthFlow. This change is consistent with the Figma designs, but I'll tag a designer for review to make sure we want to keep it.

Reply:
Also note the differentiation between `pii_from_doc` and `pii_from_user` in the two actions.

Reply:
> The ERB template uses the current flow's version of the action, so this will be called if the "Back" link is selected while using the DocAuthFlow. This change is consistent with the Figma designs, but I'll tag a designer for review to make sure we want to keep it.

But the doc auth address page doesn't call the action, it just renders the plain "Back" link to the verify step?

https://github.com/18F/identity-idp/blob/13455613f35cd77ce28235424b91402adc304ab8/app/views/idv/address/new.html.erb#L72

https://github.com/18F/identity-idp/blob/13455613f35cd77ce28235424b91402adc304ab8/app/views/idv/shared/_back.html.erb#L28-L30

https://github.com/18F/identity-idp/blob/13455613f35cd77ce28235424b91402adc304ab8/app/views/idv/shared/_back.html.erb#L16

Reply:
Good point, I think I may have confused this with a different page that was shared between the flows. I'll remove this action.

Reply:
Removed unused action.